postgresql/resource_postgresql_default_privileges.go

```diff
@@ -38,7 +38,7 @@ func resourcePostgreSQLDefaultPrivileges() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
-				Description: "Role for which apply default privileges (You can change default privileges only for objects that will be created by yourself or by roles that you are a member of)",
+				Description: "The name of an existing role of which the current (connected user) role is a member (you can change default privileges only for objects that will be created by yourself or by roles that you are a member of)",
```

Comment:
> existing role of which the current (connected user) role is a member

Hmm actually no... (thanks to this PR)

My description was indeed not very clear but the connected user does not have to be a member of the owner role as we will, with this PR, temporary grant it just to apply default privilieges.

What the owner means:

If we have a database `test` which  owner is  `test_owner` and we have another role call `readonly`, we will write:
```
resource postgresql_default_privileges "read_only_tables" {
  role     = "readonly"
  database = "test"
  schema   = "public"

  owner       = "test_owner"
  object_type = "table"
  privileges  = ["SELECT"]
}
```
So each time `test_owner` will create a table in this database (in schema `public`), `readonly` user will automatically have `SELECT` privileges on it.

But to apply this `postgresql_default_privileges`, the provider can be connected as a user (let's say `postgres`) which either is `superuser` so will have right to apply it anyway, or is not `superuser` so it will temporary grant it the `test_owner` role (if it's not already the case) to change these default privileges.

Reply:
I copied that desc from Postgres docs.. and it is correct, but in our case it is correct only for a brief moment :) We should probably describe this temp grant role process in the docs.

Reply:
It's correct for the point of view of a user executing the query. But as a user reading the provider documentation, it makes me think that the role I configured in the provider connection options needs to be a member of **owner**.
Which is not the case anymore thanks to this PR.

But maybe it's my bad English level which fails my comprehension :man_shrugging: